src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,157 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
+    private Node<T> first = null;
+    private Node<T> last = null;
+
+    public MyLinkedList() {
+    }
+
+    static class Node<T> {
```

Comment:
```suggestion
    private static class Node<T> {
```
remember about access modifiers
also move this class after pugblic methods